docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,162 @@ labels:
+This feature is available for the Tempo, Jaeger, and Zipkin data sources.
+Refer to each individual data source's documentation for configuration instructions.
 
-{{< figure src="/static/img/docs/tempo/profiles/tempo-trace-to-profile.png" max-width="900px" class="docs-image--no-shadow" alt="Selecting a link in the span queries the profile data source" >}}
+Click the document icon to open a split view in Explore with the configured data source and query relevant logs for the span.
 
-## Node graph
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-trace-to-logs.png" class="docs-image--no-shadow" max-width= "900px" caption="Trace to logs" >}}
 
-You can optionally expand the node graph for the displayed trace. Depending on the data source, this can show spans of the trace as nodes in the graph, or as some additional context like service graph based on the current trace.
+### Trace to metrics
+
+You can navigate from a span in a trace view directly to metrics relevant for that span.
+This feature is available for the Tempo, Jaeger, and Zipkin data sources.
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-node-graph.png" class="docs-image--no-shadow" max-width= "900px"  caption="Screenshot of the node graph" >}}
+Refer to each individual data source's documentation for configuration instructions.
+For Tempo, refer to [Trace to metrics configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/#trace-to-metrics).
 
-## Service Graph
+### Trace to profiles
 
-The Service Graph visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
-Once the requirements are set up, this pre-configured view is immediately available.
+Using Trace to profiles, you can use Grafana’s ability to correlate different signals by adding the functionality to link between traces and profiles.
 
-For more information, refer to the [Service Graph view section]({{< relref "../datasources/tempo/#open-the-service-graph-view" >}}) of the Tempo data source page and the [service graph view page](/docs/tempo/latest/metrics-generator/service-graph-view/) in the Tempo documentation.
+For Tempo refer to [Trace to profiles](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source#trace-to-profiles) for configuration instructions.
 
-{{< figure src="/static/img/docs/grafana-cloud/apm-overview.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the Service Graph view" >}}
+{{< figure src="/static/img/docs/tempo/profiles/tempo-trace-to-profile.png" max-width="900px" class="docs-image--no-shadow" alt="Selecting a link in the span queries the profile data source" >}}
+
+## Node graph
 
-## Data API
+You can also expand the node graph for a displayed trace. If the data source supports it, this displays spans of the trace as nodes in the graph, or provides additional context, such as a service graph based on the current trace.
```

Comment:
Might start by mentioning only some datasources support this.

Reply:
it says "if the data source supports it"

Reply:
Thanks, I did notice that. I was thinking that you should start that section noting that it might not exist. But as this has gone on long enough, I will approve.